legend-engine-config/legend-engine-repl/legend-engine-repl-client/src/main/java/org/finos/legend/engine/repl/client/Client.java

```diff
@@ -344,14 +347,12 @@ public String getLastCommand(int skip)
         }
     }
 
-    // Separate the call to System.exit() to a separate method to make it
-    // easier to mock in test, as handling System.exit() is non-trivial
-    // See https://stackoverflow.com/questions/309396/how-to-test-methods-that-call-system-exit
-    public void exit()
+    // Force exit (using System.exit()) will make sure the REPL quit completely
+    // when "exit" command is called or terminate signal with Ctrl+D/Ctrl+C in invoked
+    // otherwise, after the REPL input loop is exited, it will get stuck at a state where
+    // it still accepts input, but do nothing.
```

Comment:
I would suggest making this a proper javadoc.
```suggestion
    /**
     * Force exit using System.exit(0). This will make sure the REPL quits completely
     * when the "exit" command is called or terminate signal with Ctrl+D/Ctrl+C is invoked.
     * Otherwise, after the REPL input loop is exited, and it will get stuck at a state where
     * it still accepts input but does nothing.
     */
```

Reply:
I'll make this change in a another PR. Thank you!